src/seedu/addressbook/data/AddressBook.java

```diff
@@ -143,4 +147,12 @@ public UniquePersonList getAllPersons() {
     public UniqueTagList getAllTags() {
         return new UniqueTagList(allTags);
     }
+    
+    /**
+     * Adds changes of tags that has been made to {@link #taggings}
+     */
+    public void addChangedTag(Person person, Tag tag, boolean isAdd)
+    {
```

Comment:
the brace should come immediately after the method name, not next line.
recollect the coding standards..